Gemfile

```diff
@@ -35,7 +35,8 @@ gem 'tiny_tds'
 gem 'less-rails', git: 'https://github.com/MustafaZain/less-rails.git'
 gem 'sprockets', '3.7.1'
 gem 'execjs'
-gem 'twitter-bootstrap-rails', '~> 3.2.2'
+gem 'twitter-bootstrap-rails', '~> 4.0'
+gem 'font-awesome-rails'
```

Comment:
__[RuboCop]__

Bundler/OrderedGems: Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem `font-awesome-rails` should appear before `twitter-bootstrap-rails`.


<!-- Comment by [SideCI](https://sideci.com) -->